lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1453,6 +1463,9 @@ class IOSTextEditingStrategy extends GloballyPositionedTextEditingStrategy {
 
     _addTapListener();
 
+    // Record start time of blur subscription.
+    final DateTime blurSubscriptionStart = DateTime.now();
```

Comment:
Would be MUCH cheaper to use `StopWatch` – creating a `DateTime` object is a lot more expensive. We just need to track ticks!

Reply:
Thanks for your suggestion. I've changed to use StopWatch.